Makefile

```diff
@@ -36,11 +36,41 @@ CPPFLAGS += -DHISTFILE=\"$(HISTFILE)\" \
 OBJS += linenoise-1.0/linenoise.o
 linenoise-1.0/linenoise.o : linenoise-1.0/linenoise.c linenoise-1.0/linenoise.h
 
-## Comment these IN to enable custom baudrate support.
-## Currently works *only* with Linux (kernels > 2.6).
-#CPPFLAGS += -DUSE_CUSTOM_BAUD
-#OBJS += termios2.o
-#termios2.o : termios2.c termios2.h termbits2.h
+## Detect host machine / OS and store it into a make variable named "HOST_SYSTEM" for later use
```

Comment:
Hmm... I'd rather not have the `termios2.c` / `termios2.h` / `termbits.h` files considered, though they are *only* needed only for a specific system. I don't think the Makefile is overly complicated as shown above. A user that needs the custom baudrate support will have no problem commenting-in a couple of lines. Also: Eventually the `termios2.[ch]` / `termbits.h` stuff *should* eventually go away (I hope) once glibc is updated to support the termios2 interface...